pkg/stabilityai/main.go

```diff
@@ -0,0 +1,223 @@
+package stabilityai
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"go.uber.org/zap"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	"github.com/instill-ai/connector/pkg/base"
+	"github.com/instill-ai/connector/pkg/configLoader"
+
+	connectorPB "github.com/instill-ai/protogen-go/vdp/connector/v1alpha"
+)
+
+const (
+	venderName       = "stabilityAI"
+	host             = "https://api.stability.ai"
+	jsonMimeType     = "application/json"
+	reqTimeout       = time.Second * 60
+	textToImageTask  = "Text to Image"
+	imageToImageTask = "Image to Image"
+)
+
+//go:embed config/definitions.json
+var definitionJSON []byte
+
+var (
+	once      sync.Once
+	connector base.IConnector
+)
+
+type ConnectorOptions struct{}
+
+type Connector struct {
+	base.BaseConnector
+	options ConnectorOptions
+}
+
+type Connection struct {
+	base.BaseConnection
+	connector *Connector
+	defUid    uuid.UUID
+	config    *structpb.Struct
+}
+
+// Client represents a Stability AI client
+type Client struct {
+	APIKey     string
+	HTTPClient HTTPClient
+}
+
+// HTTPClient interface
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+func Init(logger *zap.Logger, options ConnectorOptions) base.IConnector {
+	once.Do(func() {
+		loader := configLoader.InitJSONSchema(logger)
+		connDefs, err := loader.Load(venderName, connectorPB.ConnectorType_CONNECTOR_TYPE_MODEL, definitionJSON)
```

Comment:
This has been changed to `ConnectorType_CONNECTOR_TYPE_AI`. Please update the latest connector and change the value here.